clients/cobol-lsp-vscode-extension/src/test/suite/snippet.spec.test.ts

```diff
@@ -112,3 +112,109 @@ suite.skip(
     }).timeout(helper.TEST_TIMEOUT);
   },
 );
+
+suite("TF42379 COBOL LS F96588 - Insert code snippets", function () {
+    suiteSetup(async function () {
+      this.timeout(helper.TEST_TIMEOUT);
+      helper.updateConfig("basic.json");
+      await helper.activate();
+    });
+    this.afterEach(async () => await helper.closeAllEditors()).timeout(
+      helper.TEST_TIMEOUT,
+    );
+
+    test("TC289633 Provide default COBOL code snippets - basic scenario", async () => {
+      await helper.showDocument("SNIPPET.cbl");
+      const editor = helper.get_editor("SNIPPET.cbl");
+      await helper.waitFor(() => editor.document.languageId === "cobol");
```

Comment:
Calling does not guarantee that `editor.document.languageId` is cobol at all. `waitFor` can expire and in that case it can be anything.